mifosng-android/src/main/java/com/mifos/mifosxdroid/online/notification/NotificationFetchService.java

```diff
@@ -0,0 +1,54 @@
+package com.mifos.mifosxdroid.online.notification;
+
+import android.app.IntentService;
+import android.content.Intent;
+
+import rx.Subscription;
+
+/**
+ * Created by mayankjindal on 14/08/17.
+ */
+
+public class NotificationFetchService extends IntentService {
+
+    //private final DataManager mDataManager;
+    private Subscription mSubscription;
+
+
+    public NotificationFetchService() {
+        super("NotificationFetchService");
+    }
+
+//    @Inject
+//    public NotificationFetchService(String name,
+//                                    DataManager dataManager) {
+//        super(name);
+//        mDataManager = dataManager;
```

Comment:
@therajanmaurya How can I initialize DataManager object here?
For using IntentService, I have to use the constructor with no argument. Otherwise, I get an error "intentservice has no zero argument constructor". But I want to use DataManager object for fetching notification.
And you can check my whole approach too.